features/jetpack-licensing.php

```diff
@@ -313,16 +347,21 @@ function revoke_licenses( $license_keys ) {
 			padding: 0;
 		}
 	</style>
-	<div class="jn-jetpack-products-list" style="margin: 0 0 32px 32px;">
+	<div class="jn-jetpack-products-list">
 		<label><?php esc_html_e( 'Optionally, select Jetpack products to issue licenses for:', 'jurassic-ninja' ); ?></label>
+
 		<ul>
-			<?php foreach ( $products as $slug => $label ) : ?>
-				<li>
-					<label><input type="checkbox" data-feature="jetpack-products" value="<?php echo esc_attr( $slug ); ?>"/> <?php echo esc_html( $label ); ?>
-					</label>
-				</li>
+			<?php foreach ( $families as $family ) : ?>
+				<?php foreach ( $family->products as $product ) : ?>
+					<li>
+						<label>
+							<input type="checkbox" data-feature="jetpack-products" value="<?php echo esc_attr( $product->slug ); ?>"/> <?php echo esc_html( $product->name ); ?>
+						</label>
+					</li>
+				<?php endforeach; ?>
 			<?php endforeach; ?>
 		</ul>
+
 		<label><?php esc_html_e( 'Other:', 'jurassic-ninja' ); ?></label>
 		<input type="text" data-feature="jetpack-products" value="" placeholder="<?php esc_attr_e( 'Comma-separated list of Jetpack products', 'jurassic-ninja' ); ?>" />
```

Comment:
Non-blocking, but is this "other" input necessary now that the list is fetched dynamically?

Reply:
I prefer to keep it around as a backup just in case especially considering the deploy process to JN is on the slow side which can trip us up when trying to hotfix stuff.

Reply:
Also, this allows us to test with multiple licenses of the same kind, or even poorly formatted ones, etc.

Reply:
How did I miss the multiple licenses case where I've been using that exact case as an example in our talks about the Licensing internals changes 🤣

Reply:
...and also after giving me this exact response when I too asked you about this field. You need to start taking a magnesium supplement or something.